haskell-backend/src/main/java/org/kframework/backend/haskell/HaskellKompileOptions.java

```diff
@@ -10,6 +10,6 @@ public class HaskellKompileOptions {
     @Parameter(names="--haskell-backend-command", description="Command to run the Haskell backend execution engine.", hidden = true)
     public String haskellBackendCommand = "kore-exec";
 
-    @Parameter(names="--no-haskell-binary", description="Do not force the haskell backend to use the binary format. Use the textual format instead. This is a development option. Sometime the binary format can cause issues.", hidden = true)
+    @Parameter(names="--no-haskell-binary", description="Do not force the haskell backend to use the binary format. Use the textual format instead. This is a development option. Sometime the binary format can cause issues especially on MacOS.")
```

Comment:
If we're re-writing this help message anyways, I'd prefer something like

`Use the textual KORE format in the haskell backend instead of the binary KORE format. This is a development option, but may be necessary on MacOS due to known issues with the binary format.`